_oasis

```diff
@@ -185,7 +185,7 @@ Library cli
                     CLIPager
   InternalModules:  CLIData, CLIPluginsLoaded
   CompiledObject:   byte
-  BuildDepends:     oasis, oasis.base, plugin-loader
+  BuildDepends:     oasis, oasis.base, plugin-loader, bytes
```

Comment:
the bytes library is precisely here to allow the Bytes module to be backward compatible. It's a external library available on opam